app/va/va_profile/va_profile_client.py

```diff
@@ -203,6 +291,45 @@ def get_is_communication_allowed(
         self.statsd_client.incr('clients.va-profile.get-communication-item-permission.no-permissions')
         raise CommunicationItemNotFoundException
 
+    def get_is_communication_allowed_from_profile(
+        self,
+        profile: Profile,
+        notification_type: CommunicationChannel,
+    ) -> bool:
+        """
+        Determine if communication is allowed for a given recipient, communication item, and notification type.
+
+        Args:
+            profile (Profile): The recipient's profile.
+            notification_type (CommunicationChannel): The type of the notification.
+
+        Returns:
+            bool: True if communication is allowed, False otherwise.
+
+        Raises:
+            CommunicationItemNotFoundException: If no communication permissions are found for the given parameters.
+        """
+
+        communication_permissions: CommunicationPermissions = profile.get('communicationPermissions', {})
+        for perm in communication_permissions:
+            if (
+                perm['communicationChannelName'] == notification_type.value
+                and perm['communicationItemId'] == notification_type.id
+            ):
+                self.statsd_client.incr('clients.va-profile.get-communication-item-permission.success')
+                assert isinstance(perm['allowed'], bool)
+                return perm['allowed']
+
+        self.logger.debug(
+            'V3 Profile -- did not have permission for communication item %s and channel %s',
+            notification_type.id,
+            notification_type.value,
+        )
+
+        # TODO 893 - use default communication item settings when that has been implemented
```

Comment:
I don't know what this one was; we copied this comment over from its related "v1" code

Reply:
That ticket has disappeared. Check with Kyle/Evan, but I'd remove that comment at this point.

Reply:
I think that has been covered at this point. Please remove this comment. I don't see that ticket either.